### ISSUE TITLE
compile error after commit fa3b997 for vdr-2.2.0

### ISSUE THREAD

**Issue by Karl-opec** [FernetMenta/vdr-plugin-vnsiserver#59]
compiling latest git tree leads to unkwown cStateKey timerState;

commenting line 253 in vnsitimer.c out, everything compiles and runs.

**Comment by FernetMenta** [comment]
fixed, thanks

**Comment by ralondo** [comment]
Perfect, Thanks!

**Comment by Karl-opec** [comment]
Moving 
[code]
void CVNSITimers::Action()
{
  bool modified;

#if VDRVERSNUM >= 20301
[/code]

#if VDRVERSNUM >= 20301 above   bool modified; fixes a compile warning for 2.2.0

**Comment by FernetMenta** [comment]
could you submit a pull request or at least a patch?

**Comment by Karl-opec** [comment]
Here is the patch:
[code]
--- vnsiserver/vnsitimer.c.orig 2016-07-03 19:49:12.757567734 +0200
+++ vnsiserver/vnsitimer.c      2016-07-03 19:49:42.983248570 +0200
@@ -259,9 +259,9 @@ bool CVNSITimers::IsDuplicateEvent(cTime

 void CVNSITimers::Action()
 {
-  bool modified;

-#if VDRVERSNUM >= 20301
+if VDRVERSNUM >= 20301
+  bool modified;
   cStateKey timerState;

   // set thread priority (nice level)
[/code]

**Comment by FernetMenta** [comment]
thanks, btw github markup for code snippets is 3 times this character "`" before and after the code.

```
this is code
```

**Comment by Karl-opec** [comment]
Thanks for the hint

**Comment by Karl-opec** [comment]
Please commit the following fix for a compile warning with vdr-2.2.0:

```
--- vnsitimer.c.orig    2016-07-06 10:30:09.837216870 +0200
+++ vnsitimer.c 2016-07-06 10:42:12.601092046 +0200
@@ -252,9 +252,8 @@ bool CVNSITimers::IsDuplicateEvent(cTime

 void CVNSITimers::Action()
 {
-  bool modified;
-
 #if VDRVERSNUM >= 20301
+  bool modified;
   cStateKey timerState;

   // set thread priority (nice level)
```

**Comment by FernetMenta** [comment]
done
